lib/vfscore/Config.uk

```diff
@@ -100,5 +100,19 @@ if LIBVFSCORE_AUTOMOUNT_ROOTFS
 		filesystem.
 endif
 
+config LIBVFSCORE_MAX_VOLUMES_COUNT
```

Comment:
Can we also introduce an option that switches this on or off? I am thinking of something along these lines:
```
config LIBVFSCORE_AUTOMOUNT_VOLUMES
bool "Automatically mount list of volumes from kernel arguments"
selects LIBUKLIBPARAM
default n

if LIBVFSCORE_AUTOMOUNT_VOLUMES
    config LIBVFSCORE_AUTOMOUNT_VOLUMES_COUNT
    # ...
endif
```